.ci-mgmt.yaml

```diff
@@ -34,3 +38,4 @@ actions:
       uses: webfactory/ssh-agent@v0.7.0
       with:
         ssh-private-key: ${{ secrets.PRIVATE_SSH_KEY_FOR_DIGITALOCEAN }}
+pulumiConvert: 1
```

Comment:
I'm surprised this file even exists -- I'm assuming it takes precedence over https://github.com/pulumi/ci-mgmt/blob/master/provider-ci/providers/docker/config.yaml? Ugh.

Reply:
Yes this file here is the source of truth, and if you run `make ci-mgmt` it resyncs derivative files to it.